addons/Pouches/Pouches.lua

```diff
@@ -51,11 +54,21 @@ end)
 function use_item()
     windower.chat.input('/item "' .. item.name .. '" <me>')
     item.count = item.count - 1
-    if item.count > 0 and windower.ffxi.get_player().status == 0 then
-        use_item:schedule(item.delay)
+    if item.count > 0 and windower.ffxi.get_player().status == 0 then	
+        if inventory_info.current < inventory_info.max then 
+            use_item:schedule(item.delay)
+        else
+            log("Your inventory is full...Aboriting")
+		  end
     end
 end
 
+windower.register_event('incoming chunk', function(id)
+    if id == 0x020 or 0x01E or 0x01D then
+        inventory_info.current = windower.ffxi.get_bag_info(0).count
+        inventory_info.max = windower.ffxi.get_bag_info(0).max -- this is probably not needed, but I don't want a user to have to reload when they complete a gobbie bag quest.  
```

Comment:
I think it's a good idea to keep it, but again, could save keeping track of this if you do the thang up there.